code/modules/reagents/reagents/medicine.dm

```diff
@@ -1501,3 +1501,23 @@
 	metabolism = REM * 0.002
 	overdose = REAGENTS_OVERDOSE * 0.25
 	scannable = 1
+
+/datum/reagent/earthsblood
+	name = "Earthsblood"
+	id = "earthsblood"
+	description = "A rare plant extract with immense, almost magical healing capabilities. Induces a potent psychoactive state, damaging neurons with prolonged use."
+	taste_description = "honey and sunlight"
+	reagent_state = LIQUID
+	color = "#ffb500"
+	overdose = REAGENTS_OVERDOSE * 0.50
+	scannable = 1
```

Comment:
Scannable is mostly for normal/conventional medicines. It being a more obscure, xenobotany exclusive chem doesn't make much sense to have scannable.

Reply:
I'll undo that, then. Had it scannable because of how powerful it is.